### DISCUSSION TITLE
Practice mod

### DISCUSSION THREAD

**Discussion by MightyHelper** [ppy/osu#13076] [Ideas]
**Describe the new feature:** 
This mod will show 2 cursors: the player's cursor and a cursor at 50%(maybe adjustable by something like #6272) transparency, that will move like AT, but won't do anything, The idea is to help practice fast streams or on confusing maps.
Should be unranked.

**Comment by 59xa** [comment]
Sounds like the _Finger Guides Display_ feature from **osu!stream**, would like to see this!
